src/components/atoms/CategoryButton/CategoryButton.style.ts

```diff
@@ -2,27 +2,37 @@ import { css } from '@emotion/react';
 import color from '@/styles/color';
 import typo from '@/styles/typo';
 
-export const categoryButtonBaseStyle = css`
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  width: 367px;
-  height: 189px;
-  padding: 40px 85px;
-  border-radius: 20px;
-  border: 1px solid #dedede;
-  background-color: ${color.WT};
-  box-shadow: 1px 1px 15px 0 rgba(0, 0, 0, 0.1);
-  cursor: pointer;
-  transition: all 0.3s ease-in;
-`;
+export const categoryButtonBaseStyle = css({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+  width: '367px',
+  height: '189px',
+  padding: '40px 85px',
+  borderRadius: '20px',
+  border: '1px solid #dedede',
+  backgroundColor: color.WT,
+  boxShadow: '1px 1px 15px 0 rgba(0, 0, 0, 0.1)',
+  cursor: 'pointer',
+  transition: 'all 0.3s ease-in',
+  '@media (max-width: 430px)': {
+    width: '164px',
+    height: '70px',
+    padding: '5px 50px 14px',
+    borderRadius: '7px',
+    border: '0.35px solid #dedede',
+  },
+});
 
-export const imgWrap = css`
-  flex-shrink: 0;
-`;
+export const imgWrap = css({
+  flexShrink: 0,
+});
 
-export const labelStyle = css`
-  ${typo.SubTitle}
-  margin-top: auto;
-`;
+export const labelStyle = css({
+  ...typo.SubTitle,
```

Comment:
해당 부분 역시 스프레드 연산자 대신
```suggestion
export const labelStyle = css(typo.Subtitle,{
  ...
```
으로 일관성 있는 스타일링 처리 부탁드립니다아!

Reply:
media-query 사용하는 typo에만 일괄 스프레드 연산자로 수정하였습니다..🥲 바꾸면 적용이 안되더군요...